jpo-asn-runtime/src/main/java/us/dot/its/jpo/asn/runtime/types/Asn1Bitstring.java

```diff
@@ -0,0 +1,148 @@
+package us.dot.its.jpo.asn.runtime.types;
+
+import static us.dot.its.jpo.asn.runtime.utils.BitUtils.reverseBits;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.BitSet;
+import java.util.HexFormat;
+import us.dot.its.jpo.asn.runtime.serialization.BitstringSerializer;
+
+@JsonSerialize(using = BitstringSerializer.class)
+public abstract class Asn1Bitstring implements Asn1Type {
+
+    final BitSet bits;
+    final int size;
+    final boolean hasExtensionMarker;
+    final String[] names;
+
+    public Asn1Bitstring(int size, boolean hasExtensionMarker, String[] names) {
+        this.size = size;
+        this.hasExtensionMarker = hasExtensionMarker;
+        this.bits = new BitSet(size);
+        this.names = names;
+    }
+
+    public int size() {
+        return size;
+    }
+
+    public boolean hasExtensionMarker() {

```

Comment:
question: there's a few unused methods in these classes, are they going to be used in future PRs?

Reply:
Ok this makes more sense now. Can we add some comments/documentation to indicate this kind of functionality? Even simple explanation on this extension marker could really help out ie //returns: boolean indicating whether this bitstring can be extended in the future

Reply:
Added javadocs for extension markers